lib/python/treadmill_aws/bootstrap/node/aws.linux/bin/parts/70_configure_docker.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Ensure that /var/lib/docker socket exists and Docker networking is disabled.
+# TODO: remove untrusted registry config
+
+(
+cat <<EOF
+DOCKER_NETWORK_OPTIONS=\
+  --bridge=none\
+  --ip-forward=false\
+  --ip-masq=false\
+  --iptables=false
+EOF
+) > /etc/sysconfig/docker-latest-network
+
+sed -i '17 c\
```

Comment:
i would prefer you just set entire file here, not search/replace. it is safer and easier to manage imo.